### DISCUSSION TITLE
New error: "We are unable to access the starter code on this assignment. The repository might be deleted or is not owned by ..."

### DISCUSSION THREAD

**Discussion by fuhrmanator** [community/community#17738] [GitHub Education]
For the summer semester, I "reused" an assignment that has starter code in a template (like last year's course) in the same organization, etc. Everything appeared to be working well. Students have started the assignment and there are the forked private copies of the code in their spaces in the organization.

Today, I went to check the assignment to see how many people were working, and I see the error (in a pink box at the top):

"We are unable to access the starter code on this assignment. The repository might be deleted or is not owned by (my organization)."

There is a [link](https://education.github.community/t/please-read-if-you-are-using-starter-code-outside-of-your-github-classroom-organization-for-an-assignment/66612/9) from January 2021 to explain this, but I am confused.

I'm wondering if there is some other reason for this error, since 1) it was working two days ago and 2) it seems our code is respecting the rules (it's public, even though it's outside of the organization). Did perhaps the status of the organization change? I can give details if it's necessary.

Side note - the reason I prefer to keep the starter code _outside_ the organization is that TAs (who are students) must be _owners_ of all code in the organization for a classroom (as far as I understand). When starter code is also in the organization, it means the TAs own that template, too. I would not like to have the starter code (under my control) to be theoretically modifiable by all the TAs who work (or who have worked) in the organization. We use skeleton code that is under (strict) revisions to keep the quality and simplicity, and it just doesn't make sense to have any and all TAs as "owners" of that code.

ETA: when I tried to Edit the assignment, I see that it says "Deleted repository" in the name of the starter code. I was able to re-set the correct starter code, but when I came back to the assignment some time later, it again showed the pink box with the error. I again reset the assignment starter code, and it seems stable. I will update here with more info.

**Comment by octosteve** [comment]
Hi @fuhrmanator! Do you have the name of the classroom and the assignment? I can look into it for you.
